transfermethodui/src/main/java/com/hyperwallet/android/ui/transfermethod/view/AddTransferMethodFragment.java

```diff
@@ -569,6 +589,20 @@ private boolean performValidation(boolean bypassFocusCheck) {
                             widget.showValidationError(null);
                         }
                     } else {
+                        HyperwalletInsight.getInstance().trackError(requireContext(),
+                                HyperwalletInsight.PAGE_TRANSFER_METHOD_COLLECT_ACCOUNT,
+                                HyperwalletInsight.TRANSFER_METHOD_GROUP, new ErrorInfo.ErrorInfoBuilder()
+                                        .type(HyperwalletInsight.ERROR_TYPE_FORM)
+                                        .message(widget.getErrorMessage())
+                                        .field(widget.getName())
+                                        .params(new HyperwalletInsight.TransferParamsBuilder()
```

Comment:
Sorry for the confusion, we decided to not to add additional params for this track error, please have a look on the ticket